test/async-hooks/test-internal-nexttick-default-trigger.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
Nit: style as in https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md#test-structure (and but the `//Flags` on the first line 🤷‍♂️